dotcom-rendering/src/components/GalleryImage.tsx

```diff
@@ -0,0 +1,94 @@
+import { css } from '@emotion/react';
+import { from, space, until } from '@guardian/source/foundations';
+import { grid } from '../grid';
+import { type ArticleFormat } from '../lib/articleFormat';
+import { getImage } from '../lib/image';
+import { palette } from '../palette';
+import { type ImageBlockElement } from '../types/content';
+import { GalleryCaption } from './GalleryCaption';
+import { Picture } from './Picture';
+
+type Props = {
+	format: ArticleFormat;
+	image: ImageBlockElement;
+	pageId: string;
+	webTitle: string;
+};
+
+const styles = css`
+	${grid.container}
+	grid-auto-flow: row dense;
+	column-gap: ${space[5]}px;
+
+	${until.tablet} {
+		border-top: 1px solid ${palette('--article-border')};
+		padding-top: ${space[1]}px;
+	}
+
+	${from.tablet} {
+		&::before {
+			${grid.between('grid-start', 'centre-column-start')}
+			grid-row: span 2;
+			content: '';
+			background-color: ${palette('--article-background')};
+			border-right: 1px solid ${palette('--article-border')};
+		}
+
+		&::after {
+			${grid.between('centre-column-end', 'grid-end')}
+			grid-row: span 2;
+			content: '';
+			background-color: ${palette('--article-background')};
+			border-left: 1px solid ${palette('--article-border')};
+		}
+	}
+
+	${from.desktop} {
+		&::after {
+			${grid.between('right-column-end', 'grid-end')}
+		}
+	}
+
+	${from.leftCol} {
+		&::before {
+			${grid.between('grid-start', 'left-column-start')}
+		}
+	}
```

Comment:
If we use `grid.paddedContainer` we may be able to replace all of this with:

```suggestion
	${from.tablet} {
		border-left: 1px solid ${palette('--article-border')};
		border-right: 1px solid ${palette('--article-border')};
	}
```